app/services/create_fund_activity.rb

```diff
@@ -20,9 +20,7 @@ def call
     activity.accountable_organisation_reference = "GB-GOV-13"
     activity.accountable_organisation_type = "10"
 
-    activity.extending_organisation_name = "Department for Business, Energy and Industrial Strategy"
-    activity.extending_organisation_reference = "GB-GOV-13"
-    activity.extending_organisation_type = "10"
+    activity.extending_organisation = Organisation.find_by!(service_owner: true)
```

Comment:
Agreed. I don't really like how this is modelled for the resons you mention. I wonder if we have a similar issue with HMT, we could keep HMT hard wired but that seems restrictive but we do know that HMT role in the service is pretty limited.

I started thinking about organisation 'roles' and a table (Pundit policy?) of who can do what e.g.

HMT
Fund level: Can fund, Cannot Extend, Cannot be accountable, cannot impliment
Programme: level: none
Project level: none
…

Delivery Partner(in gov)
Fund level: none
Programme level: Can extend
Project level: Can fund, can extend, cannot be accountable, cannot impliment
…

This could let us find delivery partners by asking for 'organisations that can extend projects', BEIS by asking 'organisaiton that can fund programmes, etc and could give more flexibility as thing change but could also be too complicated.

Reply:
We talked this though at the white board - we agreed we would keep the simple boolean for `service_owner` and let the refactor come as it is required rather than pre-empt it.